routes/survey.js

```diff
@@ -169,10 +173,10 @@ function join(profileName, current, compare, previous, previousCompare) {
   for (let i = 0; i < current.length; i++) { // eslint-disable-line
     const row = current[i];
     const { id, variable, variablename, base, category, profile } = row;
-    const rowConfig = find(specialCalculationConfigs[profileName], ['variable', row.variable]);
-    const compareRow = compare.find(compare => compare.id === row.id);
-    const previousRow = previous.find(previous => previous.id === row.id);
-    const previousCompareRow = previousCompare.find(previousCompare => previousCompare.id === row.id);
+    const rowConfig = find(specialCalculationConfigs[survey], ['variable', row.variable]);
+    const compareRow = compare.find(c => c.id === row.id);
+    const previousRow = previous.find(p => p.id === row.id);
+    const previousCompareRow = previousCompare.find(p => p.id === row.id);
 
     const difference = doDifferenceCalculations(row, compareRow);
```

Comment:
Wait you added a new `isDecennial` parameter to these three functions, but didn't pass in an argument to that parameter....

Reply:
This was why the difference calculations weren't showing up